packages/transport/src/transports/bridge.ts

```diff
@@ -269,25 +269,28 @@ export class BridgeTransport extends AbstractTransport {
     }
 
     public send({ session, name, data, protocol }: AbstractTransportMethodParams<'send'>) {
-        return this.scheduleAction(async signal => {
-            const { encode } = protocol || bridgeProtocol;
-            const bytes = buildMessage({
-                messages: this.messages,
-                name,
-                data,
-                encode,
-            });
-            const response = await this.post('/post', {
-                params: session,
-                body: bytes.toString('hex'),
-                signal,
-            });
-            if (!response.success) {
-                return response;
-            }
+        return this.scheduleAction(
+            async signal => {
+                const { encode } = protocol || bridgeProtocol;
+                const bytes = buildMessage({
+                    messages: this.messages,
+                    name,
+                    data,
+                    encode,
+                });
+                const response = await this.post('/post', {
+                    params: session,
+                    body: bytes.toString('hex'),
+                    signal,
+                });
+                if (!response.success) {
+                    return response;
+                }
 
-            return this.success(undefined);
-        });
+                return this.success(undefined);
+            },
+            { timeout: undefined },
```

Comment:
but this is post, not read/received as stated in PR description?

Reply:
lol, sorry my bad! fixed, forcepushed